astropy/cosmology/constants.py

```diff
@@ -0,0 +1,17 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Cosmology-related constants.
+
+This module and its contents adhere to the `Cosmology API`_.
+"""
+
+import astropy.constants as _consts
+
+__all__ = [
+    "G",  # Gravitational constant
+    "c",  # Speed of light
+]
+
+
+G = _consts.G.to("pc km2 s-2 Msun-1")
```

Comment:
You're probably aware but just in case: these will not be instances of `Constant`, but rather `Quantity`.

Reply:
Having both `astropy.constants` and `astropy.cosmology.constants` is too confusing for me.